package.json

```diff
@@ -23,13 +23,15 @@
     "yargs": "^17.5.1"
   },
   "devDependencies": {
+    "@aws-sdk/client-lambda": "^3.213.0",
```

Comment:
We only installed this so it would be bundled in the tests, correct?

Reply:
yep, exactly.